lib/arclight/traject/ead2_config.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+require 'traject'
+require 'traject/nokogiri_reader'
+require 'traject_plus'
+require 'traject_plus/macros'
+require 'arclight/normalized_date'
+require 'arclight/normalized_title'
+require 'active_support/core_ext/array/wrap'
+require 'arclight/digital_object'
+require 'arclight/year_range'
+
+# rubocop:disable Style/MixinUsage
+extend TrajectPlus::Macros
+# rubocop:enable Style/MixinUsage
+
+settings do
+  provide 'nokogiri.namespaces',
+          'xmlns' => 'urn:isbn:1-931666-22-9'
```

Comment:
This is not blocking and only out of curiosity, but I'm unfamiliar with  the patterns used in `traject`.  Should `urn:isbn:1-931666-22-9` be structured as constant in https://github.com/projectblacklight/arclight/blob/master/lib/arclight/custom_document.rb or https://github.com/projectblacklight/arclight/blob/master/lib/arclight.rb?

Reply:
I'm inclined to leave it out for now. Our approach to namespaces might change so I'd like to leave open the flexibility of that possibility until we have more established conventions. 🤷‍♂